recipe/meta.yaml

```diff
@@ -50,6 +51,7 @@ outputs:
         - python-dateutil >=2.8.2
         - sqlglot >=10.4.3
         - toolz >=0.11
+        - xxhash >=0.8
```

Comment:
Should these be here or in the `pooch` recipe? Ibis doesn't explicitly depend on these, we just pass them as extras.